### PR TITLE
Login with credentials

### PR DESCRIPTION
**Pull request by TUNESHMAN**
# Description

Describe change or new feature here.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

Remove any items which are not applicable.

- [ ] I have performed a self-review of my own code
- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works .
- [ ] New and existing unit tests pass locally with my changes

## Link to Trello card

### CODE REVIEW

File: src/state/IsLoggedIn/IsLoggedInActionCreators.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import axios from "axios";
+import * as types from "./isLoggedInActionTypes";
+
+const action = (type, payload) => (payload ? {type, payload} : {type});
+
+export const login = (errorSetter, loginData) => async dispatch => {
+  dispatch(action(types.LOGIN_START));
+  try {
+    const response = await axios.post("/api/auth/register", loginData);
```
Comment (Darragh1996) [inline src/state/IsLoggedIn/IsLoggedInActionCreators.js]: Need to update axios request endpoint

File: src/pages/Login.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        )}
+      </Form.Item>
+      <Form.Item>
+        {getFieldDecorator("password", {
+          //rules are for the form validation
+          rules: [
+            {required: true, message: "Please input a password!"},
+            {
+              type: "string",
+              message: "Invalid password",
```
Comment (niklasbec) [inline src/pages/Login.js]: needs min length of 5 characters

File: src/pages/Login.css [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  }
+  .ant-form-explain {
+      color: red;
+  }
+  .forgot-password h1{
+      margin: 0 auto;
+      text-align: center;
+      margin-bottom: 30px;
+      font-size: 2rem;
+  }
```
Comment (niklasbec) [inline src/pages/Login.css]: Form not centered

File: src/pages/Login.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+  const [error, setError] = useState("");
+  const handleChange = e => {
+    setFormValues({
+      ...formValues,
+      [e.target.name]: e.target.value,
+    });
+  };
+  const handleSubmit = e => {
+    e.preventDefault();
+    console.log(props.login);
```
Comment (niklasbec) [inline src/pages/Login.js]: code needs to be removed

File: src/pages/Login.js [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+    props.login(setError, formValues);
+  };
```
Comment (niklasbec) [inline src/pages/Login.js]: no axios call

Comment (sergeikabuldzhanov) [reply src/pages/Login.js]: It's inside an action

**Pull request merged**